doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -16,7 +16,7 @@ Overview
 The Modem SLM library allows you to perform the following functions:
 
 * Manage the serial interface so that the application only decides which UART device to use and configures its DTS.
-* Manage the GPIO pins, with support for bidirectional indication and wakeup.
+* Manage the GPIO pins, with support for bidirectional indication, and power pin toggling.
```

Comment:
```suggestion
* Manage the GPIO pins, with support for bidirectional indication and power pin toggling.
```

Reply:
I used comma to make a distinction on where toggling refers to. It should refer only to power pin and not indication pin.
Does it mean that without the comma?

Reply:
I'll remove the word toggling from the end after tinkering a bit.